$NetBSD: patch-scripts_w3mman_w3mman2html.cgi.in,v 1.1 2015/08/24 13:42:28 leot Exp $

Use of defined() on aggregates (arrays and hashes) is deprecated from Perl 5.22.

--- scripts/w3mman/w3mman2html.cgi.in.orig	2011-01-04 09:22:28.000000000 +0000
+++ scripts/w3mman/w3mman2html.cgi.in
@@ -220,7 +220,7 @@ sub is_command {
   local($p);
 
   (! -d && -x) || return 0;
-  if (! defined(%PATH)) {
+  if (! %PATH) {
     for $p (split(":", $ENV{'PATH'})) {
       $p =~ s@/+$@@;
       $PATH{$p} = 1;
